### PR TITLE
fix: renamed IHRC to IHRC719

### LINKED ISSUE #807: Rename IHRC to IHRC-719
### Problem

Currently, the IHRC719 is stored in IHRC.sol file. This should be updated to IHRC719.sol to be more descriptive. Moreover, new IHRC632 is being added. Renaming IHRC.sol to IHRC-719 will make it better to distinguish the two IHRCs.

### Solution

Rename IHRC.sol to IHRC-719.sol

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**: renamed IHRC contracts to IHRC719 along with its related artifacts

**Related issue(s)**:

Fixes #807 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: contracts-abi/contracts/hts-precompile/IHederaAccountService.sol/IHederaAccountService.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+[
```
Comment (acuarica) [inline contracts-abi/contracts/hts-precompile/IHederaAccountService.sol/IHederaAccountService.json:1]: Is this file needed? If yes, is it needed here? I don't see where is it used anywhere in the PR.

Comment (quiet-node) [reply contracts-abi/contracts/hts-precompile/IHederaAccountService.sol/IHederaAccountService.json:1]: Ah good catch but yes it's the artifact for the IHederaAccountService interface Luke forgot to check in.

**Review by acuarica — APPROVED** [review]
lgtm
